rhel8/profiles/pci-dss.profile

```diff
@@ -3,7 +3,7 @@ documentation_complete: true
 title: 'PCI-DSS v3.2.1 Control Baseline for Red Hat Enterprise Linux 8'
 
 description: |-
-    Ensures PCI-DSS v3.2.1 related security configuration settings are applied.
+    Ensures PCI-DSS v3.2.1-related security configuration settings are applied.
```

Comment:
@ggbecker please just remove `-related`. Or, keep the Americanized version.

Reply:
I think removing `-related` will be beneficial both upstream/downstream.

PR updated.

Reply:
@ggbecker yes it is. Thanks very much!